Tests/Marketplace/print_test_playbook_summary.py

```diff
@@ -59,7 +59,11 @@ def print_test_playbooks_summary(artifacts_path: Path, without_jira: bool) -> bo
     test_playbooks_report = artifacts_path / TEST_PLAYBOOKS_REPORT_FILE_NAME
 
     # iterate over the artifacts path and find all the test playbook result files
-    test_playbooks_result_files_list = get_test_results_files(artifacts_path, TEST_PLAYBOOKS_REPORT_FILE_NAME)
+    if not (test_playbooks_result_files_list := get_test_results_files(artifacts_path, TEST_PLAYBOOKS_REPORT_FILE_NAME)):
+        # Write an empty report file to avoid failing the build artifacts collection.
+        JUnitXml().write(test_playbooks_report.as_posix(), pretty=True)
+        logging.error(f"Could not find any test playbook result files in {artifacts_path}")
```

Comment:
More intuitive order of these commands.
```suggestion
        logging.error(f"Could not find any test playbook result files in {artifacts_path}, writing an empty report file")
        # Write an empty report file to avoid failing the build artifacts collection.
        JUnitXml().write(test_playbooks_report.as_posix(), pretty=True)
```